src/pg/relations.cpp

```diff
@@ -62,4 +70,23 @@ void EstimateRelSize(Relation rel, int32_t *attr_widths, BlockNumber *pages, dou
 	PostgresFunctionGuard(estimate_rel_size, rel, attr_widths, pages, tuples, allvisfrac);
 }
 
+Oid
+GetRelidFromSchemaAndTable(const char *schema_name, const char *entry_name) {
+	List *name_list = NIL;
+	name_list = lappend(name_list, makeString(pstrdup(schema_name)));
+	name_list = lappend(name_list, makeString(pstrdup(entry_name)));
+	RangeVar *table_range_var = makeRangeVarFromNameList(name_list);
+	return RangeVarGetRelid(table_range_var, AccessShareLock, true);
```

Comment:
We should probably wrap this whole function in a PostgresFunctionGuard, or maybe introduce a new template/macro to be able to guard a block instead of a function call, or guard a function definition instead of a function call.

Reply:
> guard a function definition instead of a function call.

Yeah I have that somewhere, I'll dig it up and update it. In the meantime I've wrapped the whole function call.